framework_tvb/tvb/interfaces/web/templates/jinja2/burst/main_burst.html

```diff
@@ -53,7 +53,7 @@ <h4><mark>Parameter Space Exploration</mark> <span id="title-visualizers"></span
 	
 	<!--! Configure + View portlets for selected Burst -->
 	<section id="section-portlets" class="view-portlets view-column col-3">
-		<h4><mark>Visualize</mark> <span id="title-visualizers">New simulation</span></h4>
+		<h4><mark>Visualize</mark> <span id="section-title-visualizers">New simulation</span></h4>
```

Comment:
I think we should keep the previous name.
It was ok the "duplication" because it was either one or another section to display, and both should change name on selected burst switch

Reply:
If we keep it like this the title in for the Visualise section will never update.

Reply:
Why not? I also changed the JS?
Have you tested after a clear browser cache ?